examples/basecoin/tests/cli/basictx.sh

```diff
@@ -116,5 +116,9 @@ test03CreditTx() {
 
 # Load common then run these tests with shunit2!
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )" #get this files directory
-. $DIR/common.sh
-. $DIR/shunit2
+
+# TODO: how to handle this if we are not in the same directory
+CLI_DIR=${DIR}/../../../../tests/cli
```

Comment:
The $DIR command navigates to a directory relative to THIS bash file... So this shouldn't be an issue, alternatively (maybe more clearly) we could just make $DIR a function of the $GOPATH

Reply:
CLI_DIR=$GOPATH/src/github.com/cosmos/cosmos-sdk/tests/cli

Reply:
Looks better to me

Reply:
Maybe in a different repo, we would refer to the cosmos-sdk in vendor dir